### PR TITLE
Resolving #283 update dependent packages

### PR DESCRIPTION
**Pull request by huanlin**
There are 3 components left for update in the future:

- LiteDB
- Swashbuckle (updated to v4)
- Microsoft.Extensions.Logging 

Note for LiteDB: when update from v4 to v5, we might need to add `Connection=Shared` and `Upgrade=true` in connection string. (Some changes are preserved in branch [f283-update-dependent-packages-bak](https://github.com/huanlin/FlubuCore/tree/f283-update-dependent-packages-bak))

### CODE REVIEW

**Comment by huanlin** [comment]
> I have noticed Analyzer tests are not run:
> 
> > Executing task DotnetTestTask
> > Running program from 'C:\projects\flubucore\src':

ok, I'll see what happened.

**Comment by huanlin** [comment]
> Unable to find C:\projects\flubucore\src\FlubuCore.Analyzers.Tests\bin\Release\netcoreapp2.0\testhost.dll. Please publish your test project and retry.

Turned out the test project has to target .NET Core 2.1 or 3.x.
Ref: https://stackoverflow.com/questions/54770830/unable-to-find-testhost-dll-please-publish-your-test-project-and-retry

I noticed that there are lots of differences in bin/[config]/FlubuCore.Analyzers.Tests.deps.json between netcoreapp2.0 and netcoreapp2.1. Not sure whether it's relevant or not.

File: src/FlubuCore.Analyzers.Tests/FlubuCore.Analyzers.Tests.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```
Comment (huanlin) [inline src/FlubuCore.Analyzers.Tests/FlubuCore.Analyzers.Tests.csproj:4]: Turned out the test project has to target .NET Core 2.1 or 3.x.
Ref: https://stackoverflow.com/questions/54770830/unable-to-find-testhost-dll-please-publish-your-test-project-and-retry

I noticed that there are lots of differences in bin/[config]/FlubuCore.Analyzers.Tests.deps.json between netcoreapp2.0 and netcoreapp2.1. Not sure whether it's relevant or not.

**Comment by mzorec** [comment]
Everything looks ok now. Thank you for yet another PR :)

**Pull request merged**